proxy/src/main/java/com/velocitypowered/proxy/connection/client/ClientPlaySessionHandler.java

```diff
@@ -629,4 +638,8 @@ public void flushQueuedMessages() {
       }
     }
   }
+
+  private boolean isAllowedCharacter(char c) {
```

Comment:
Nice catch! Could you document this function noting the source (apart from the client source) like wiki.vg for the allowed characters?